assets/js/custom/Translation.js

```diff
@@ -2,6 +2,137 @@
 
 // eslint-disable-next-line no-unused-vars
 function Translation (hasDescription, hasCredit) {
+  const providerMap = {
+    'itranslate': 'iTranslate'
+  }
+  let languageMap = {}
+  let targetLanguage = document.documentElement.lang
+  let creditText = 'Translated by %provider% from %sourceLanguage% to %targetLanguage%'
+
+  $(document).ready(function () {
+    setTargetLanguage()
+    getLanguageMap()
+    getTranslationCreditText()
+  })
+
+  $(document).on('click', '.comment-translation-button', function () {
+    const commentId = $(this).attr('id').substring('comment-translation-button-'.length)
+
+    $(this).hide()
+
+    if (isTranslationAvailable(commentId)) {

```

Comment:
I think the condition might have been inversed here. When I read `isTranslationAvailable`, i think then we don't need to translate again and we can directly open the translated text. So here it would be `if (!isTranslationAvailable())`

Reply:
Changed helper to isTranslationNotAvailable